documentation/js-sdk-guide/modules/tooling-guide/pages/card-generation.adoc

```diff
@@ -25,7 +25,7 @@ A schematics will be generated using the following command:
 ng generate [collection-name]:[schematics-name]
 
 
-For demonstration purposes, the https://github.com/eclipse-esmf/esmf-aspect-model-editor/blob/main/core/apps/ame/src/assets/aspect-models/org.esmf.examples.movement/1.0.0/Movement.ttl[Movement.ttl,window=_blank,opts=nofollow] Aspect Model will be used.
```

Comment:
Why `window=_blank` is removed here? It should open a new window since it is not part of the documentation